package/crds/lambda.aws.crossplane.io_functions.yaml

```diff
@@ -447,6 +440,21 @@ spec:
                     description: The reason code for the last update that was performed
                       on the function.
                     type: string
+                  layers:
+                    description: The function's layers (https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html).
+                    items:
+                      properties:
+                        arn:
+                          type: string
+                        codeSize:
+                          format: int64
+                          type: integer
+                        signingJobARN:
+                          type: string
+                        signingProfileVersionARN:
+                          type: string
+                      type: object
+                    type: array
```

Comment:
I think this is a breaking API change too.

Reply:
it is only in status filed - so i think this is no breaking api change